test/integration/etcd_storage_path_test.go

```diff
@@ -0,0 +1,1010 @@
+	unversioned.GroupVersionResource{Group: "extensions", Version: "v1beta1", Resource: "replicationcontrollerdummies"}: {ephemeral: true}, // not stored in etcd
+	unversioned.GroupVersionResource{Group: "extensions", Version: "v1beta1", Resource: "jobs"}:                         {ephemeral: true}, // creating this makes a apisbatchv1.Job so test that instead
+	unversioned.GroupVersionResource{Group: "extensions", Version: "v1beta1", Resource: "scales"}:                       {ephemeral: true}, // not stored in etcd, part of kapiv1.ReplicationController
+	unversioned.GroupVersionResource{Group: "extensions", Version: "v1beta1", Resource: "thirdpartyresourcedatas"}:      {ephemeral: true}, // we cannot create this  // TODO but we should be able to create it in kube
+	// --
+
+	// k8s.io/kubernetes/pkg/apis/imagepolicy/v1alpha1
+	unversioned.GroupVersionResource{Group: "imagepolicy.k8s.io", Version: "v1alpha1", Resource: "imagereviews"}: {ephemeral: true}, // not stored in etcd
+	// --
+
+	// k8s.io/kubernetes/pkg/apis/policy/v1beta1
+	unversioned.GroupVersionResource{Group: "policy", Version: "v1beta1", Resource: "poddisruptionbudgets"}: {
+		stub:             `{"metadata": {"name": "pdb1"}, "spec": {"selector": {"matchLabels": {"anokkey": "anokvalue"}}}}`,
+		expectedEtcdPath: "kubernetes.io/poddisruptionbudgets/etcdstoragepathtestnamespace/pdb1",
+	},
+	unversioned.GroupVersionResource{Group: "policy", Version: "v1beta1", Resource: "evictions"}: {ephemeral: true}, // not stored in etcd, deals with evicting kapiv1.Pod
+	// --
+
+	// k8s.io/kubernetes/pkg/apis/rbac/v1alpha1
+
+	// we cannot create these  // TODO but we should be able to create them in kube
+	unversioned.GroupVersionResource{Group: "rbac.authorization.k8s.io", Version: "v1alpha1", Resource: "roles"}:               {ephemeral: true},
+	unversioned.GroupVersionResource{Group: "rbac.authorization.k8s.io", Version: "v1alpha1", Resource: "clusterroles"}:        {ephemeral: true},
+	unversioned.GroupVersionResource{Group: "rbac.authorization.k8s.io", Version: "v1alpha1", Resource: "rolebindings"}:        {ephemeral: true},
+	unversioned.GroupVersionResource{Group: "rbac.authorization.k8s.io", Version: "v1alpha1", Resource: "clusterrolebindings"}: {ephemeral: true},
+	// --
+
+	// k8s.io/kubernetes/pkg/apis/storage/v1beta1
+	unversioned.GroupVersionResource{Group: "storage.k8s.io", Version: "v1beta1", Resource: "storageclasses"}: {
+		stub:             `{"metadata": {"name": "sc1"}, "provisioner": "aws"}`,
+		expectedEtcdPath: "kubernetes.io/storageclasses/sc1",
+	},
+}
+
+// Only add objects to this list when there is no mapping from GVK to GVR (and thus there is no way to create the object)
+var gvkWhiteList = createGVKWhiteList(
+	// k8s.io/kubernetes/pkg/api/v1
+	unversioned.GroupVersionKind{Group: "", Version: "v1", Kind: "DeleteOptions"},
```

Comment:
I'd also be ok whitelisting these kinds regardless of group/version... this will be a pain to maintain as we get lots of groups